zerver/views/typing.py

```diff
@@ -38,13 +38,13 @@ def send_notification_backend(
 
     if recipient_type_name == "stream":
         if stream_id is None:
-            raise JsonableError(_("Missing stream_id"))
+            raise JsonableError(_("Missing channel ID"))
```

Comment:
This might be able to be standardized to match what `REQ` would do if `stream_id` hadn't been passed and that wasn't a required parameter; seems like a useless specialized string.

Reply:
Sounds good. Updated to use the same `REQ` error string: `"Missing '{var_name}' argument"`.